packages/sdk-components-react/src/markdown-embed.tsx

```diff
@@ -0,0 +1,22 @@
+import { micromark } from "micromark";
+import { forwardRef, useMemo } from "react";
+
+type MarkdownEmbedProps = {
+  code: string;
+  className?: string;
+  // avoid builder passing it to dom
+  children?: never;
+};
+
+export const MarkdownEmbed = /* @__PURE__ */ forwardRef<
```

Comment:
we prob need to do that @pure thing everywhere

Reply:
Yes, though would not give us much. Especially when radix is used.

Reply:
what if we had a lint rule to add it whenever necessary? https://classic.yarnpkg.com/en/package/eslint-plugin-better-tree-shaking

Reply:
I mean we can't change radix packages. We could try but they will not accept this because of worse DX without displayName. The only way is to wait til react 19 is released and then wait til radix will switch to it.